src/plugins/cargo-workspace.ts

```diff
@@ -285,6 +282,24 @@ export class CargoWorkspace extends WorkspacePlugin<CrateInfo> {
     };
   }
 
+  protected postProcessCandidates(
+    candidates: CandidateReleasePullRequest[],
+    updatedVersions: VersionsMap
+  ): CandidateReleasePullRequest[] {
+    const root_candidate = candidates.find(c => c.path === ROOT_PROJECT_PATH);
+    if (!root_candidate) {
+      throw Error('Unable to find root candidate pull request');
+    }
+
+    root_candidate.pullRequest.updates.push({
+      path: 'Cargo.lock',
+      createIfMissing: false,
+      updater: new CargoLock(updatedVersions),
+    });
```

Comment:
nit: let's keep our variable names lowerCamelCased. `root_candidate` -> `rootCandidate`

Reply:
Done, obviously I was still in rust mode when I wrote this :)